Update note for Example 2 (#8441)
MicrosoftDocs/PowerShell-Docs: reference/7.2/Microsoft.PowerShell.Management/Copy-Item.md

>
> `Copy-Item -Path "C:\Logfiles" -Destination "C:\Drawings" -Recurse`

> [!NOTE]

> > if the path `C:\Drawings` does not exist the result of the above will be to copy all the files from the `Logfiles` folder into a single file `Drawings` in the root of C:\
> 

Reviewer: ```suggestion
> If the path `C:\Drawings` does not exist the cmdlet copies all the files from the `Logfiles` folder into a single file `C:\Drawings`.

```
